releasenotes/notes/deprecate_b3_propagation_style_for_b3multi-d8c31144253a1b39.yaml

```
@@ -0,0 +1,4 @@
+---
+deprecations:
+  - |
+    propagation: Configuration of propagation style with ``DD_TRACE_PROPAGATION_STYLE=b3`` is deprecated and will be removed in version 2.0.0. Please use the newly added ``DD_TRACE_PROPAGATION_STYLE=b3multi`` instead. This change adopts the values used for OpenTelemetry propagators.
```

```suggestion
    propagation: Configuration of propagation style with ``DD_TRACE_PROPAGATION_STYLE=b3`` is deprecated and will be removed in version 2.0.0. Please use the newly added ``DD_TRACE_PROPAGATION_STYLE=b3multi`` instead.
```

I think having this extra bit implies that there were functional changes as well? Which this is just a name change only.